client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCheckout.tsx

```diff
@@ -22,62 +24,64 @@ const stripePromise = loadStripe(pk);
+        formData.seatingAcc = formData.comments;
+      }
+      const donation = formData.donation;
+      const stripe = await stripePromise;
+      if (!stripe) return;
+      const response = await fetch(
+        process.env.REACT_APP_API_2_URL + `/events/checkout`,
+        {
+          credentials: 'include',
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({cartItems, formData, donation, discount}),
         },
-        body: JSON.stringify({cartItems, formData, donation, discount}),
-      },
-    );
-    const session = await response.json();
-    const paymentIntent = session.payment_intent;
-    const result = await stripe.redirectToCheckout({sessionId: session.id});
-    if (result.error) {
-      console.error(result.error.message);
+      );
+      if (!response.ok) {
+        throw response;
+      }
+      const session = await response.json();
+      if (session.id === 'comp') {
+        dispatch(removeAllTicketsFromCart());
+        navigate(`/success`);
+        // return <CheckoutSuccess />;
+      }
+      const paymentIntent = session.payment_intent;
+      const result = await stripe.redirectToCheckout({sessionId: session.id});
+      if (result.error) {
+        console.error(result.error.message);
+      }
+    } catch (error) {
+      console.error('Error response status: ', error.status);
+      if (error.json) {
+        const errorMessage = await error.json();
+        console.error('Error message from server: ', errorMessage);
+      }
     }
   };
 
   return (
     <div className='w-full h-screen overflow-x-hidden absolute'>
       <div className='flex flex-col lg:ml-[15rem] lg:mx-[5rem] md:ml-[13rem] tab:mx-[2rem] mx-[0.5rem] mt=[5rem] mb-[9rem]'>
-        <div className='flex flex-row items-center h-auto mt-[7.25rem] md:w-full mb-5'>
-          <button
-            onClick={() => navigate('../ticketing/purchaseticket')}
-            className='bg-blue-500 hover:bg-blue-600 px-3 py-2 rounded-xl flex flex-row items-center text-zinc-100'
-          >
-            <svg
-              xmlns='http://www.w3.org/2000/svg'
-              className='h-5 w-5'
-              viewBox='0 0 20 20'
-              fill='currentColor'
-            >
-              <path
-                fillRule='evenodd'
-                d='M9.707 14.707a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 1.414L7.414 9H15a1 1 0 110 2H7.414l2.293 2.293a1 1 0 010 1.414z'
-                clipRule='evenodd'
-              />
-            </svg>
-            back to Purchase
-          </button>
-        </div>
+        <div className='flex flex-row items-center h-auto mt-[7.25rem] md:w-full mb-5'></div>
         <div className='flex flex-row items-center mt-2 text-zinc-800'></div>
         <div className='flex flex-col items-center md:flex-row rounded-[1rem] md:items-stretch md:bg-white sm:flex-col w-full h-full'>
```

Comment:
When the 2 divs are removed, this div will need the `mt-[7.5rem]`

Reply:
add mt-[6rem] to div
![image](https://github.com/WonderTix/WonderTix/assets/97830813/3ccf144d-0a56-4c8d-8bf3-7584968edc62)